README.md

```diff
@@ -89,17 +89,15 @@ end
 println("Undiscounted reward was $rsum.")
 ```
 
-For more examples with visualization, see the documentation below and [POMDPGallery.jl](https://github.com/JuliaPOMDP/POMDPGallery.jl).
+For more examples and examples with visualizations, reference the [Examples](https://JuliaPOMDP.github.io/POMDPs.jl/latest/examples) and [Gallery of POMDPs.jl Problems](https://JuliaPOMDP.github.io/POMDPs.jl/latest/gallery) sections of the documentaiton.
```

Comment:
I think there is a better way to make these links relative - maybe [Examples](/examples) or something. We don't really want to always point to latest

Reply:
I know you can make relative links in the documentation. I wasn't aware you could do it from the README.md to the documentation. I'll look into that. I agree that it would be the better way to go.